source/write/update.txt

```diff
@@ -0,0 +1,229 @@
+.. _php-write-update:
+
+================
+Update Documents
+================
+
+.. contents:: On this page
+   :local:
+   :backlinks: none
+   :depth: 2
+   :class: singlecol
+
+.. facet::
+   :name: genre
+   :values: reference
+
+.. meta::
+   :keywords: modify, change, bulk, code example
+
+Overview
+--------
+
+In this guide, you can learn how to use the {+php-library+} to update
+documents in a MongoDB collection. You can call the ``MongoDB\Collection::updateOne()``
+method to update a single document or the ``MongoDB\Collection::updateMany()``
+method to update multiple documents.
+
+Sample Data
+~~~~~~~~~~~
+
+The examples in this guide use the ``restaurants`` collection in the ``sample_restaurants``
+database from the :atlas:`Atlas sample datasets </sample-data>`. To access this collection
+from your PHP application, instantiate a ``MongoDB\Client`` that connects to an Atlas cluster
+and assign the following value to your ``collection`` variable:
+
+.. literalinclude:: /includes/write/update.php
+    :language: php
+    :dedent:
+    :start-after: start-db-coll
+    :end-before: end-db-coll
+
+To learn how to create a free MongoDB Atlas cluster and load the sample datasets, see the
+:atlas:`Get Started with Atlas </getting-started>` guide.
+
+Update Operations
+-----------------
+
+You can perform update operations in MongoDB by using the following methods:
+
+- ``MongoDB\Collection::updateOne()``, which updates *the first document* that
+  matches the search criteria
+- ``MongoDB\Collection::updateMany()``, which updates *all documents* that match
+  the search criteria
+
+Each update method requires the following parameters:
+
+- **Query filter** document: Specifies which documents to update. For
+  more information about query filters, see the 
+  :manual:`Query Filter Documents section </core/document/#query-filter-documents>` in
+  the {+mdb-server+} manual.
+
+- **Update** document: Specifies the **update operator**, or the kind of update to
+  perform, and the fields and values to change. For a list of update operators
+  and their usage, see the :manual:`Field Update Operators guide
+  </reference/operator/update-field/>` in the {+mdb-server+} manual.
+
+Update One Document
+~~~~~~~~~~~~~~~~~~~
+
+The following example uses the ``updateOne()`` method to update the ``name``
```

Comment:
Q: Do we need to refer to this with the full `MongoDB\Collection::updateOne()` name every time? There's a few other occurrences in this PR where we don't use the long method name as well.

Reply:
I've been using the full name for first mentions in each top-level section, then the shortened name for future mentions in that section and in its subsections.

Reply:
Makes sense to me and looks like this page adheres to that rule.